assets/js/base/components/block-error-boundary/style.scss

```diff
@@ -1,15 +1,16 @@
 .wc-block-components-error {
 	display: flex;
-	background-color: #f3f3f4;
-	border-left: 4px solid #6d6d6d;
+	background-color: $gray-100;
+	border-left: 4px solid $gray-300;
 	padding: $gap-larger $gap;
 	align-items: center;
 	justify-content: center;
 	flex-direction: column;
+	color: $gray-700;
 }
 
 .wc-block-components-error__header {
-	@include font-size(larger);
+	@include font-size( larger );
```

Comment:
Values in parentheses shouldn't be padded according to the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values)

Reply:
Prettier should catch this @ralucaStan so might be another IDE config issue.